### PR TITLE
test: Add kcli to test the scripts

### PR DESCRIPTION
**Pull request by qinqon**
Using kcli [1]  is possible to test cmdline + ignition (patched) + MC.
This change add a kcli directory with scripts to start it up.

### CODE REVIEW

**Comment by qinqon** [comment]
@phoracek @RamLavi

**Review by phoracek — COMMENTED** [review]
Just a nit. The guide looks nice. I did not manage to verify it yet, `[root@nike13 rhcos-slb]# ./kcli/run.sh ./kcli/ocp.yaml openshift-pull.json` times out for me

File: kcli/ocp.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    nets: 
+    - name: default
+      mac: 52:54:00:f6:80:03
+    - name: default
+      mac: 52:54:00:f6:80:04
+
+rhocs-slb:
+ type: kube
+ kubetype: openshift
+ domain: qinqon.com
```
[the remark this answers is not in the document]
Comment (qinqon) [reply kcli/ocp.yaml]: Done

**Review by phoracek — APPROVED** [review]
Thanks

**Comment by phoracek** [comment]
The test result is unrelated to this change

**Pull request merged**